doc/source/glossary.rst

```diff
@@ -95,3 +95,12 @@ Glossary
     UIUC
       The University of Illinois at Urbana-Champaign
 
+    TKE
```

Comment:
When content complete these should be placed in the correct alphabetical order with the other terms

Reply:
Looks fixed